scripts/config-parser/bin/dhcp_config_parser.rb

```diff
@@ -0,0 +1,9 @@
+#!/bin/ruby
+
+require_relative "../lib/dhcp_config_parser"
+
+reservation_diff = File.open("./data/reservation_diff.json", "w")
+
+reservation_diff <<
+  DhcpConfigParser.run.to_json
```

Comment:
I would have this on one line. Looks a little odd otherwise

```suggestion
reservation_diff << DhcpConfigParser.run.to_json
```